drivers/pinmux/stm32/pinmux_stm32f4.h

```diff
@@ -42,9 +42,11 @@
 
 #define STM32F4_PINMUX_FUNC_PA11_USART6_TX  STM32_PINMUX_ALT_FUNC_8 | STM32_PUSHPULL_PULLUP
 #define STM32F4_PINMUX_FUNC_PA11_UART4_RX   STM32_PINMUX_ALT_FUNC_11 | STM32_PUSHPULL_PULLUP
+#define STM32F4_PINMUX_FUNC_PA11_OTG_FS_DM  STM32_PINMUX_ALT_FUNC_10 | STM32_PUSHPULL_NOPULL
 
 #define STM32F4_PINMUX_FUNC_PA12_USART6_RX  STM32_PINMUX_ALT_FUNC_8 | STM32_PUSHPULL_PULLUP
 #define STM32F4_PINMUX_FUNC_PA12_UART4_TX   STM32_PINMUX_ALT_FUNC_11 | STM32_PUSHPULL_PULLUP
+#define STM32F4_PINMUX_FUNC_PA12_OTG_FS_DP  STM32_PINMUX_ALT_FUNC_10 | STM32_PUSHPULL_NOPULL
```

Comment:
We need to add () around complex macros.

Reply:
thanks, done. Seems we sill need to update all pinmux files as well.